mpqp/tools/maths.py

```diff
@@ -229,7 +237,7 @@ def exp(angle: Expr | Complex) -> sp.Expr | complex:
 
 
 def rand_orthogonal_matrix(
-    size: int, seed: Optional[int] = None
+    size: int, seed: Optional[Union[int, np.random.Generator]] = None
```

Comment:
same thing, inform what happens when seed is not entered

Reply:
But here Hamza, they are not the same!
The second specifies that the variable can hold either an int, a np.random.Generator (a generator for random numbers from the NumPy library), or be None.
?

Reply:
Ok, ut the question is the same, what happens if the parameter is None ?

Reply:
Then this should appear in the documentation so the user knows. This is the same remark as above, and everywhere else we have ``seed`` appearing in the doc string